docs/blog/newsletter-february-2018.rst

```diff
@@ -0,0 +1,121 @@
+
+If you're planning on coming to the conference, but you're worried about persuading your company to cover the costs, 
+you might be interested in `our new Convince Your Manager page <http://www.writethedocs.org/conf/portland/2018/convince-your-manager/>`_. 
+We've knocked together an example email and some resources that you can use to make your case.
+
+And finally, in non-conference news, we're thrilled to announce that the **Write the Docs Job Board** will be launching next week. 
+One of the benefits of the WTD community has always been its ability to connect documentarians with employers 
+who understand the value of good docs. The job board will be a destination for making those connections. 
+Keep an eye on Slack, Twitter, the forum, your email, etc. – we'll let you know as soon as it goes live.
+
+Whew! Told you there's a lot going on! But now, let's have a look at a few of the conversations that have been happening
+in `Slackland <http://www.writethedocs.org/slack/>`_ over the new year.
+
+********************************
+It's not personal, it's feedback
+********************************
+
+Have you ever needed to tell someone that their writing is... bad? 
+How do you manage that conversation without being discouraging or unkind? 
+Many in the community have been in this position and this month, WTD documentarians had great tips 
+for sharing feedback effectively and constructively:
+
+* **Follow the Toastmasters' PIPS method: praise, improvement, praise, summary.** Start with something they did well, explain what they can do to improve, move on to more praise, and then summarize. Including the praise helps your feedback land without damaging their confidence.
+* **Keep your feedback specific and results-oriented, and don't forget the 'why'.** It's helpful to couch your feedback in terms of the reader's experience. Try something like, "I think users could misinterpret this to mean X instead of Y. Revising to Z would help prevent confusion."
+* **Follow company culture.** Depending on your team's culture, folks might be fine with receiving a marked-up file of revisions, or they might expect to receive feedback during a collaborative discussion. Do your best to stick to established channels.
+* **Refer to outside sources.** Use tools like style guides or checklists of common problems to handle proofreading and standardization issues objectively.
+
+*Want to chat more about constructive criticism? Head to* `#general <https://writethedocs.slack.com/messages/general>`_ *on Slack.*
+
+***************************************
+Questions to ask during a job interview
+***************************************
+
+A good job interview always comes down to the questions asked. 
+But that responsibility falls not only to the interview*er*, but also to the interview*ee*. 
+Last month, WTDers had some excellent suggestions for how to ask intelligent, engaged questions at the end of a job interview:
```

Comment:
WTDers? Really?

Reply:
This is the only one I feel strongly about.

Reply:
😜 fixed